text/src/autogluon/text/automm/data/process_label.py

```diff
@@ -0,0 +1,34 @@
+from typing import Optional, List, Any, Union
+from nptyping import NDArray
+from ..constants import LABEL
+from .collator import Stack
+
+
+class LabelProcessor:
```

Comment:
Docstrings added. This is where label customization goes since we want to fuse any combinations of models. Currently, we don't do label customization for each model. But in future this will provide us the flexibility to do so. For now, you can think this as the abstract label processing class.